ui-library-add-call-recording/api/types.cs

```diff
@@ -0,0 +1,7 @@
+namespace Contoso
+{
+    class Result
+    {
+        public string text { get; set; }
```

Comment:
properties should be UpperCamelCase

Reply:
Thanks for pointing that out. This particular type is gone in the very next PR, so letting it be.